q-and-a.md

```diff
@@ -0,0 +1,48 @@
+
+## Hoe is Nuts ontstaan?
+
+Nuts is ontstaan vanuit een gedeelde realisatie onder software leveranciers: we zijn veel tijd kwijt met talloze maatwerk koppelingen te bouwen tussen elkaars systemen. Deze koppelingen zijn meestal kostbaar, tijdrovend en foutgevoelig. Hierdoor is het vandaag de dag voor veel zorgverleners niet goed mogelijk om digitaal samen te werken met collega's die gebruik maken van een ander ICT-systeem. In 2018 hebben deze leveranciers de handen in een geslagen en zijn het Nuts initiatief gestart. Dit geen geen praatgroepje maar een bouwgroep!
+
+## Waarom is een open communicatienetwerk in de zorg belangrijk?
+
+Veel zorg gaat over zorginstellingen heen. Het is een complexe dialoog tussen specialisten, de patiënt, mantelzorg, langdurige zorg etc. Al deze partijen werken het liefst in hun eigen systemen maar hebben wel informatie van elkaar nodig. Door een open communicatienetwerk te introduceren maken we dit mogelijk. Vanuit je eigen software op een uniforme wijze informatie uitwisselen en communiceren met zorgverleners van je eigen of andere organisaties.
+
+Het is belangrijk dat dit netwerk een open netwerk wordt omdat iedereen mee moet kunnen doen: zowel de grote ziekenhuizen als de kleine zorgonderneming met 4 clienten. Door rekening te houden met zowel grote als kleine organisaties denken we na over zowel capaciteit als hele lage kosten voor deelname. Zo blijft het netwerk voor iedereen beschikbaar.
+Bekijk ons [manifest](/manifest) op onze website voor de andere uitgangspunten.
+
+## Waarom zouden softwareleveranciers zich bij jullie moeten aansluiten?
+
+ICT leveranciers in de zorg staan allemaal voor dezelfde uitdaging: het leggen van een digitaal lijntje met elke andere leverancier. Deze koppelingen leggen we nu vaak één-op-één tussen twee leveranciers, wat kostbaar, tijdrovend en foutgevoelig is. Een softwarepakket onderhoudt al snel tien tot honderd verschillende koppelingen met andere leveranciers. Op basis van de Nuts uitgangspunten zal er een gemeenschappelijke technische basis ontstaan waardoor het voor leveranciers veel makkelijker en goedkoper wordt om een koppeling te realiseren. Daardoor kan er meer energie gestoken worden in belangrijkere zaken, zoals een goede procesondersteuning. Dat resulteert uiteindelijk in veel betere producten waarmee de zorg beter kan samenwerken.
+
+## Wat is het verdienmodel?
+
+Nuts is een samenwerkingsverband van softwareleveranciers en heeft daarom ook geen verdienmodel. Door samen te werken aan een gedeelde infrastructuur pakt elke leverancier een deel van de rekening op. Dit laat zich terugbetalen in een betere concurrentiepositie. Immers een betere informatievoorziening is voor de zorgprofessional een belangrijk punt waarop zij hun keuze voor leverancier baseren.
```

Comment:
Dit laat zich terugbetalen...

Dit laat zich terugbetalen in lagere kosten voor het bouwen en beheren van koppelingen. Ook verbeter je jouw positie als leverancier door betere procesondersteuning voor jouw gebruikers te bieden. Er is uiteraard wel een kostenmodel, aangezien iedere aangesloten partij de eigen Nuts node zal beheren. 

[laatste zin kan weg]